content/en/docs/k8s-operator/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: OpenTelemetry Operator for Kubernetes
+linkTitle: K8s Operator
+weight: 11
+description:
+  An implementation of a Kubernetes Operator, that
+  manages collectors and auto-instrumentation of the workload using OpenTelemetry
+  instrumentation libraries.
+spelling: cSpell:ignore Otel
+aliases: [/docs/operator]
+---
+
+## Introduction
+
+The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https://coreos.com/operators/).
+
+The operator manages:
+
+- OpenTelemetry Collector
+- auto-instrumentation of the workloads using OpenTelemetry instrumentation libraries
+
+## Getting started
+
+To install the operator in an existing cluster, make sure you have cert-manager 
```

Comment:
This is a common source of confusion. Perhaps we should note that this is not 100% necessary, if people know where to place the certs?

Reply:
> To get started quickly, make sure you have the cert-manager ... Note that the cert-manager isn't required by the operator, as long as the requested certificates are made available in the right places.

The actual documentation should list what those right places are. I don't have them from the top of my head, but I know that the webhook endpoints use those certs.

cc @yuriolisa, @pavolloffay, @frzifus

Reply:
The operator uses admission webhooks that require secure communication between the API server and the webhook. The official operator installation reqiures cert-manager for provisioning TLS certificates. 
However, the certificates can be provided manually as well. For instance, [operator lifecycle manager](https://github.com/operator-framework/operator-lifecycle-manager) can be used to deploy the operator without the dependency on the cert-manager.

Reply:
> The official operator installation reqiures cert-manager for provisioning TLS certificates

The operator has dependency on the certs, not on cert-manager, right? There's no hard dependency on the operator and the APIs provided by cert-manager, from what I remember.

Reply:
>The operator has dependency on the certs, not on cert-manager, right? There's no hard dependency on the operator and the APIs provided by cert-manager, from what I remember.

thats correct.

Reply:
Same comment here: the goal of this PR is to have a starting point, I am OK to have this issue fixed right now, but I would prefer to merge this PR and then work through incremental changes to improve the docs for the operator :)

Reply:
> The operator has dependency on the certs, not on cert-manager, right?

yes, sort of. 

The official installation manifest use cert-manager that provisions the certificates. We don't document or directly support people deploying without cert-manager in the otel-operator community (Let's take OLM separatelly).

Reply:
just trying to give this a close, @pavolloffay: can we keep the doc as it is or should we remove some of that section and point to the README for now only?

Reply:
yes. I have already approved the PR, other changes can go in follow up PRs

Reply:
Ack.